### ISSUE TITLE
Play with terraform graph

### ISSUE THREAD

**Issue by dragan** [scipian/quoins#65]
For scipian/community#35, we're going to look into using [`terraform graph`](https://www.terraform.io/docs/commands/graph.html) to generate images of what each quoin creates and include it in our documentation. It should also help flesh out issues with configuration, when depends_on is not set correctly.

Once this spike is complete, please create the corresponding stories in the epic and include any others that I have missed and estimate them.

**Comment by nicklathe** [comment]
[WIP] Using `terraform graph`
- [ ] General usage of terraform graph
- [ ] Usage of graphviz
- [ ] Implement terraform graph with our quions

**Comment by nicklathe** [comment]
@dragan do we want a separate story for each quoin (etcd, vault, rethinkdb, kubernetes) of one story that tracks all of these as subtasks?

**Comment by nicklathe** [comment]
Here's an example of the Kubernetes graph. It's very large. ![graph](https://user-images.githubusercontent.com/8847422/31914518-db29d764-b7ff-11e7-8acd-cdd9871f822f.png)

**Comment by nicklathe** [comment]
The vault graph is smaller and more readable: ![graph](https://user-images.githubusercontent.com/8847422/31914677-7a467c58-b800-11e7-87e8-8d158f809110.png)

**Comment by nicklathe** [comment]
Lastly, an example of Kubernetes with the size and ratio flags used. It helps keep things from getting too squashed: 
![graph](https://user-images.githubusercontent.com/8847422/31914803-f6f2676c-b800-11e7-9fa1-7d2570b9e847.png)

**Comment by dragan** [comment]
Let's create a single story for now and then put each quoin as a sub-task.

**Comment by nicklathe** [comment]
Added #66 to epic, as an outcome of this spike.
